venv/CalculationClass.py

```diff
@@ -0,0 +1,50 @@
+import pandas as pd
+
+
+#Query Analysis Calculation Class
+class Calculations():
```

Comment:
```py
class Calculations:
```
Remove Brackets.

Remove comment. Give class a more expressive name so you don't need the comment.